reflex-dom.cabal

```diff
@@ -75,6 +76,7 @@ library
     Reflex.Dom.Builder.Class
     Reflex.Dom.Builder.Class.Events
     Reflex.Dom.Builder.Immediate
+    Reflex.Dom.Builder.Static
```

Comment:
In addition to adding the module here, it should also be added to Reflex.Dom (which re-exports pretty much everything).